client/src/pages/reports/Compact.js

```diff
@@ -725,7 +725,7 @@ const CompactViewHeader = ({
   return (
     <header css={HEADER_STYLE}>
       <h3 css={HEADER_TITLE_STYLE} value="title">
-        {Settings.fields.report.compactView}
+        {Settings.fields.report.compactView || "Summary / Print"}
```

Comment:
These defaults introduce repeated identical strings. I suggest to either make these fiends required in the schema or to have another mechanism dealing with the defaults